testsuite/tests/singlecluster/limitador/section/test_listener.py

```diff
@@ -0,0 +1,24 @@
+"""Tests that the RLP is correctly applies to the chosen Gateway Listener"""
+
+import pytest
+
+from testsuite.kuadrant.policy.rate_limit import Limit, RateLimitPolicy
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
+
+
+@pytest.fixture(scope="module")
+def rate_limit(cluster, blame, module_label, gateway):
+    """Add a RateLimitPolicy targeting the specific Gateway Listener"""
+    rlp = RateLimitPolicy.create_instance(cluster, blame("limit"), gateway, "api", labels={"testRun": module_label})
```

Comment:
Please use `GatewayListener.name` if referencing the default name for listener `api`

Reply:
I think it is better for the section name to be explicitly written here